packages/framework/src/Resources/views/Common/Inline/Icon/icon.html.twig

```diff
@@ -0,0 +1,8 @@
+{% if type == 'svg-font' %}
+    <i class="svg svg-{{ name }} {{ attr.class|default('')}}"
+        {% if attr.title|default('') %}title ="{{ attr.title }}"{% endif %}
+        {% for key, value in attr.data|default('') %}
+            data-{{ key }} ="{{ value }}"
```

Comment:
I belive the space is not intentional here.

```suggestion
            data-{{ key }}="{{ value }}"
```

Reply:
fixed

Reply:
Cool!